src/functions/Stock.js

```diff
@@ -31,6 +34,10 @@ const formatResponse = stock => {
   };
 };
 
+const formatValues = value => {
```

Comment:
You could update the function name to `formatValue` because it only formats one value